test/1_stdlib/TestData.swift

```diff
@@ -127,9 +127,11 @@ class TestData : TestDataSuper {
     // String of course has its own way to get data, but this way tests our own data struct
     func dataFrom(_ string : String) -> Data {
         // Create a Data out of those bytes
-        return string.nulTerminatedUTF8.withUnsafeBufferPointer { (ptr) in
-            // Subtract 1 so we don't get the null terminator byte. This matches NSString behavior.
-            return Data(bytes: ptr.baseAddress!, count: ptr.count - 1)
+        return string.utf8CString.withUnsafeBufferPointer { (ptr) in
+            ptr.baseAddress!.withMemoryRebound(to: UInt8.self, capacity: ptr.count) {
```

Comment:
Annoyingly, `Data(bytes: UnsafeRawPointer, count: Int)` isn't available in this repo as it now is in swift-corelibs-foundation. Should I address that in this PR too?